### ISSUE TITLE
[help] Scannning github to find all the repos with a particular word

### ISSUE THREAD

**Issue by ojasviashishchauhan** [N0MoreSecr3ts/wraith#65]
Is there a way to scan entire GitHub to find all the repo/code which contains a specific word?

**Comment by mattyjones** [comment]
@ojaschauhan44 You cannot scan all of GitHub very easily as rate limiting will catch up to you sooner or later. Also at best you could only scan all public repos. In order to do this slow enough that rate limits didn't mess you up and you don't get blocked as a suspected DOS attack the length of time would be unrealistic.

If you managed a way around the rate limits and still kept up a reasonable performance metric you will most likely be identified as a suspected DDOS event. If you can get around that and you crank this thing up to 11, when it is completed, you will run into similar issues that masscan faces, which include the ability to knock network gear over.

Long story short, no you would need to do massive scans like that against local targets.
